jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/MetricAssertions.java

```diff
@@ -136,4 +153,19 @@ private static void assertAttributedPoints(
                             KeyValue::getKey, keyValue -> keyValue.getValue().getStringValue())))
         .satisfiesExactlyInAnyOrder(assertions);
   }
+
+  @SuppressWarnings("unchecked")
+  private static void assertAttributedMultiplePoints(
```

Comment:
[for reviewer] the difference with `assertAttributedPoints` is that we check all the provided assertions to all the data points, whereas `assertAttributedPoints` expects to have exactly one assertion per point.